<?xml version="1.0" encoding="utf-8"?><testsuites><testsuite errors="1" failures="0" hostname="LAPTOP-CGROQ9OG" name="pytest" skipped="0" tests="1" time="2.749" timestamp="2020-07-04T11:09:41.267032"><testcase classname="" file="test_main.py" name="test_main" time="0.000"><error message="collection failure">test_main.py:1: in &lt;module&gt;
    from main import *
main.py:31: in &lt;module&gt;
    get_ipython().run_line_magic(&apos;matplotlib&apos;, &apos;inline&apos;)
E   NameError: name &apos;get_ipython&apos; is not defined</error></testcase></testsuite></testsuites>